lib/MetaCPAN/Document/Favorite.pm

```diff
@@ -41,7 +41,7 @@ Sets the C<_timestamp> field to the value of L</date>.
 
 has timestamp => (
     is        => 'ro',
-    timestamp => { path => 'date', store => 1 },
+    timestamp => {}, # { path => 'date', store => 1 },
```

Comment:
This is good.  We'd just need to add the timestamp manually.  If we don't do that in this pull request, we should open an issue for it so that we don't forget.